src/mk.sh

```diff
@@ -610,6 +610,9 @@ function mk_build()
     esac
   done
 
+  # sets default values for any missing configurations in .config
+  cmd_manager "$flag" "make olddefconfig"
```

Comment:
Single quote around the command.

Also, `cmd_manager` will show this command; I think we can set the default flag to SILENT here. Finally, you need to check if there is a `.config` file before running this command; if there is no config file, do not run this command.